### DISCUSSION TITLE
PT微调后的模型如何增加每次输出的最大长度

### DISCUSSION THREAD

**Discussion by liuxingpeng520521** [zai-org/ChatGLM3#538] [Q&A]
PT微调后的模型如何本地部署，使用api调用？可以给一个demo吗

**Comment by yuanye1010** [comment]
> PT微调后的模型如何本地部署，使用api调用？可以给一个demo吗

用官方给的inference.py 改一下模型的路径和微调出的pt路径就行了

**Comment by zRzRzRzRzRzRzR** [comment]
#253 在这里讨论
